Dispatcher/UI/Screens/UserPage/UserPageViewModel.swift

```diff
@@ -0,0 +1,4 @@
+import Foundation
```

Comment:
do not create uneeded files which add overhead
and impact project compilation times
remove untill needed